Content.Server/GameObjects/Components/Mobs/StunnableComponent.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.Threading;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces.GameObjects;
+using Content.Server.Mobs;
+using Content.Shared.Audio;
+using Content.Shared.GameObjects.Components.Mobs;
+using Robust.Server.GameObjects;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Timers;
+using Robust.Shared.IoC;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+using Timer = Robust.Shared.Timers.Timer;
+
+namespace Content.Server.GameObjects.Components.Mobs
+{
+    [RegisterComponent]
+    public class StunnableComponent : Component, IActionBlocker, IAttackHand
+    {
+        [Dependency] private IEntitySystemManager _entitySystemManager;
+        [Dependency] private ITimerManager _timerManager;
+
+        private bool _stunned = false;
```

Comment:
Why have this, rather than check if `_stunnedTimer > 0`?

Reply:
Yeah, those are useless now that I removed timers. Didn't think about removing them at first, though. Removed now.